tensorflow/compiler/jit/xla_cluster_util.cc

```diff
@@ -224,13 +225,9 @@ struct XlaGlobalJitLevel {
 };
 
 XlaGlobalJitLevel GetXlaGlobalJitLevel(
-    const GraphOptimizationPassOptions& options) {
```

Comment:
I changed this interface a bit as `GraphOptimizationPassOptions` is not accessible in meta_optimizer.cc.